components/formatted-markdown.js

```diff
@@ -0,0 +1,20 @@
+// Documentation for markdown-to-jsx
+// https://github.com/probablyup/markdown-to-jsx
+import React from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl'
+import Markdown from 'markdown-to-jsx'
+
+const FormattedMarkdown = ({ intl, id }) => (
+  // <Markdown source={intl.formatMessage({ id })} />
+  <Markdown>
```

Comment:
We may want to make use of https://www.npmjs.com/package/markdown-to-jsx#optionsoverrides---override-any-html-tags-representation so that the styling from our components (for example `<Heading>` and `<Link>`) are used in rendered markdown as well.

On design.ooni.io I was using https://github.com/mdx-js/mdx with a babel loader, but we probably don't want to do that here as supporting jsx in markdown here is not wise as translators could inject malicious JS code and we don't have the need for it.

Reply:
I looked into @mdx-js/mdx, but it seemed more useful for 'JSX in Markdown' which isn't our usecase. So it was between `react-markdown` and `markdown-in-jsx`. This one claimed to have a smaller bundle size. Took their word for it and used it.

Reply:
Removed the commented line from when I was evaluating [`react-markdown`](https://github.com/rexxars/react-markdown).

Reply:
Did you look into if the styling is going to look consistent in the markdown rendered content?